### DISCUSSION TITLE
Download speed is slow.

### DISCUSSION THREAD

**Discussion by Tanggue** [aclap-dev/video-downloadhelper#210] [Help and Q&A]
### Your question

After the update, the download speed became very slow.
The download speed is 100-300kb, so it takes 5 hours to download a 1-hour video.
Why has my speed suddenly slowed down? Is there a solution?

I'm using it with Mozilla Firefox.

### Addon & CoApp version and details

Version 8.2.1.1 (stable)
Mozilla Firefox 123.0

**Comment by paulrouget** [comment]
We need to know which website you're downloading from.

And please do what it says [there](https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F) and copy the result here.
